upstream_sync.py

```diff
@@ -24,10 +24,24 @@
 tmp_dir = '/var/tmp/upstream_sync-{0}'.format(user)
 
 
-def build_yum_config(name, url, sslcacert, sslcert, sslkey):
+def make_dir(dir_path, mode=0700):
+    """ checks if a directory exists and create it if necessary
+    :param mode: mode for makedirs
+    :param dir_path: dir path for check
+
+    :return: None
+    """
+    if not os.path.isdir(dir_path):
+        try:
+            os.makedirs(dir_path, mode)
+        except IOError as e:
+            print e
+            sys.exit(1)
+
+
+def build_yum_config(name, url, sslcacert, sslcert, sslkey, exclude):
     # Check tmp path exist
-    if not os.path.isdir(tmp_dir):
-        os.mkdir(tmp_dir)
+    make_dir(tmp_dir)
```

Comment:
This changes the code behavior. In the previous code, the users umask would be used, but now directories will be created with 0700. Was this done intentionally, if so what is the reasoning? There shouldn't be any sensitive information in the generated repo file, so I think I would prefer to default to the users umask.